index.html

```diff
@@ -86,6 +86,18 @@ <h3 class="ba-skill__title">dreams.</h3>
         </div>
 
     </main>
+    <footer>
+        <div class="footer__wraper">
+            <form action="">
+                <input type="text" placeholder="Emaile">
+                <button>Submit</button>
+            </form>
+            <ul>
+                <li><a href="mailto:example@gmail.com">example@gmail.com</a></li>
+                <li><a href="tel:+380974737123">+380974737123</a></li>
```

Comment:
I hope that this phone number is not real.

Reply:
Sure,  this is a non-existent number